packages/icon/src/lib/Icons/Actions.tsx

```diff
@@ -57,3 +60,11 @@ export const ChartColumnIcon = ({ ...props }: SvgIconProps): JSX.Element => (
 export const ExpandIcon = ({ ...props }: SvgIconProps): JSX.Element => <FaSvgIcon icon={faAngleDown} {...props} />;
```

Comment:
Since we have 2 now can we also change the name of plain expand icon to something like AngleExpand or ChevronExpand if you want to be fancy.